### DISCUSSION TITLE
How to make wireless speaker to play music? Because I want to make a community broadcasting system.

### DISCUSSION THREAD

**Discussion by HenryDu8133** [cc-tweaked/CC-Tweaked#1895] [Q&A] [answered]
How to make wireless speaker to play music? Because I want to make a community broadcasting system.
It needs to be wireless and can be uniform control by a computer?.
I found that I can't connect computer to speaker with wireless modem only by wire. This causes the server to be stuck and unsightly .
All in all. I need your help .Please give me the detailed procedure and scheme .Thanks!

**Comment by zyxkad** [comment]
You can't, you must control peripherals by computer through wires.

But, you can place computers, which have wireless modems, next to the speakers. And let your center controller to control the computers under the speakers

**Reply by HenryDu8133** [reply]
Thanks, I also thought about it, but I don't know how to use center computer to control the computers near the speaker. Can you give further instructions?

**Reply by zyxkad** [reply, marked as answer]
There is a [rednet API](https://tweaked.cc/module/rednet.html) to make you control modems easily.

Basic logic is: player give audio data to controller -> controller broadcast audio data to clusters -> cluster plays the audio on the speaker

**Reply by HenryDu8133** [reply]
Alright , I'll have a try.Thank you very much !🌹
